web-comp.js

```diff
@@ -0,0 +1,204 @@
+const catalogProducts = [
+    {
+        id: 1,
```

Comment:
Again my/chatgpt fault.
Get used to use strings for ids and not numbers.
Also they must not start with numbers, add a prefix ex: `product_<pprogressiveNumber>`

Reply:
fixed